src/components/Reports/SubscriberHistory/SubscriberHistory.test.js

```diff
@@ -1,23 +1,219 @@
 import React from 'react';
-import { render, cleanup } from '@testing-library/react';
+import { render, cleanup, wait } from '@testing-library/react';
```

Comment:
Why did you add this new `wait` here? it was not here before.

Reply:
Forget this comment, this was marked as change in all files view, but in this commit it has no changes.

Reply:
because in the old file before rename dont have and now yes in the new file